### PR TITLE
using a script for insert the date to the database

### PR DESCRIPTION
**Pull request by ameentalahmeh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "localStart": "nodemon ./server/server.js",
     "build": "cd client && npm run build",
     "dev": "npm run client && npm run localStart",
-    "heroku-postbuild": "cd client && npm install && npm run build"
+    "heroku-postbuild": "cd client && npm install && npm run build",
+    "db:query": "psql -U staff -d ydrc1 ./queries.sql"
```
Comment (shahenazmonia) [inline package.json]: You have to add file path after symbol ```-f``` 
The script should be ```psql -U 'database_username' -d 'database_name' -a -f 'file_path' ```

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
+insert into users (username, password, role)
+  values ('Admin','$2a$10$D/IX/AtYw5YHT4YWI2B2aOg5ZGMBodHNDx2x6vIbHWomyH4fsJ9SG','admin');
+insert into users (username, password, role)
+  values ('ydrc','$2a$10$D/IX/AtYw5YHT4YWI2B2aOg5ZGMBodHNDx2x6vIbHWomyH4fsJ9SG','user');
+
+insert into events (title,start_date,end_date,org_name,price,capacity,note,"userId",status)
+  values ('Interview',DATE 'Thu, 14 February 2019 13:30:00', DATE 'Thu, 14 February 2019 13:30:00','YDRC',120,10,'The event is for the interview',0,1);
```
Comment (shahenazmonia) [inline queries.sql]: Here's ```userId=0```  Ids are starting counting from 1 not from 0

File: package.json (hunk 1 above)
Comment (ameentalahmeh) [reply package.json]: done :+1:

File: queries.sql (hunk 2 above)
Comment (ameentalahmeh) [reply queries.sql]: done :+1:

**Pull request merged**